test/integration/spec/general/login_test.rb

```diff
@@ -4,7 +4,7 @@ class SignInTest < PoltergeistIntegrationTest
 
   describe 'user' do
     it 'can login' do
-      puts "Hello, world!"
+      puts 'Hello, world!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.